cmd/glasskube/cmd/uninstall.go

```diff
@@ -66,14 +67,18 @@ var uninstallCmd = &cobra.Command{
 			}
 		}
 
+		if uninstallCmdOptions.DryRun {
+			fmt.Fprintln(os.Stderr, "🔎 Dry-run mode is enabled. Nothing will be changed.")
+		}
+
```

Comment:
Please move this so that it is shown before the uninstall details and confirmation prompt